src/Main.java

```diff
@@ -0,0 +1,25 @@
+public class Main {
+
+    public static void main(String[] args) {
+        //Create a method that will check if a given string is a palindrome.
+        // If it is it should print “This string is a palindrome”
+        // if not print “This string is not a palindrome”
+        String original = "racecar";
+        String reverse = "";
+        for (int i = original.length() - 1; i >= 0; i--) {
+            reverse += original.charAt(i);
+        }
+        boolean palindrome = false;
+        for (int i = 0; i < original.length(); i++) {
```

Comment:
Your program does not work! Your code will return "This string is a palindrome" in all cases because you just assign 'true' value to palindrome variable. You need to compare two strings, and if original string and reverse string are same it will be a palindrome